hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEnv.java

```diff
@@ -65,40 +68,72 @@ public HapiTestEnv(@NonNull final String testName, final boolean cluster) {
             final var configText = sb.toString();
 
             for (int nodeId = 0; nodeId < numNodes; nodeId++) {
-                final var workingDir = Path.of("./build/hapi-test/" + testName + "/node" + nodeId)
-                        .normalize();
+                final var workingDir =
+                        Path.of("./build/hapi-test/node" + nodeId).normalize();
                 setupWorkingDirectory(workingDir, configText);
-                if (nodeId == 0) {
-                    nodes.add(new InProcessHapiTestNode(workingDir, 0, FIRST_GRPC_PORT));
+                final var nodeName = NODE_NAMES[nodeId];
+                final var acct = AccountID.newBuilder().accountNum(3L + nodeId).build();
+                if (useInProcessAlice && nodeId == 0) {
+                    nodes.add(new InProcessHapiTestNode(nodeName, nodeId, acct, workingDir, FIRST_GRPC_PORT));
                 } else {
-                    nodes.add(new SubProcessHapiTestNode(workingDir, nodeId, FIRST_GRPC_PORT + (nodeId * 2)));
+                    nodes.add(new SubProcessHapiTestNode(
+                            nodeName, nodeId, acct, workingDir, FIRST_GRPC_PORT + (nodeId * 2)));
                 }
             }
         } catch (Exception e) {
             throw new RuntimeException(e);
         }
     }
 
+    /**
+     * Starts all nodes in the environment.
+     */
     public void start() {
+        started = true;
         for (final var node : nodes) {
             node.start();
         }
 
         for (final var node : nodes) {
-            node.waitForActive(60);
+            try {
+                node.waitForActive(60);
```

Comment:
This needs to be changed to a larger value (as done in a previously merged PR) constant or keyed off of the `secondsWaitingServerUp` property you're setting elsewhere in this PR.
Otherwise the servers simply do not have enough time to reach an ACTIVE state in the CI environment.